src/BDCS/API/V0.hs

```diff
@@ -1287,6 +1274,54 @@ projectsList pool moffset mlimit = do
     apply_limits l = take limit $ drop offset l
 
 
+-- | The JSON response for /projects/info
+data ProjectsInfoResponse = ProjectsInfoResponse {
+    pipProjects :: [Projects]                                   -- ^ List of recipe names
+} deriving (Show, Eq)
+
+instance ToJSON ProjectsInfoResponse where
+  toJSON ProjectsInfoResponse{..} = object [
+      "projects" .= pipProjects ]
+
+instance FromJSON ProjectsInfoResponse where
+  parseJSON = withObject "/projects/info response" $ \o -> do
+    pipProjects <- o .: "projects"
+    return ProjectsInfoResponse{..}
+
+-- | /api/v0/projects/info/<projects>
+-- Return information about the comma-separated list of projects
+--
+-- # Example
+--
+-- > {
+-- >     "projects": [
+-- >         {
+-- >             "description": "The GNU tar program saves many files ...",
+-- >             "homepage": "http://www.gnu.org/software/tar/",
+-- >             "name": "tar",
+-- >             "summary": "A GNU file archiving program",
+-- >             "upstream_vcs": "UPSTREAM_VCS"
+-- >         }
+-- >     ]
+-- > }
+-- >
+projectsInfo :: ConnectionPool -> String -> Handler ProjectsInfoResponse
+projectsInfo pool project_names = do
+    let project_name_list = map T.pack $ sortBy caseInsensitive $ argify [project_names]
+    projects_info <- liftIO $ mapMaybeM getProjectInfo project_name_list
+    return $ ProjectsInfoResponse projects_info
+  where
+    getProjectInfo :: T.Text -> IO (Maybe Projects)
+    getProjectInfo project_name = do
+        result <- runExceptT $ flip runSqlPool pool $ do
+            mproj_id <- findProject project_name
+            case mproj_id of
+                Nothing      -> return Nothing
+                Just proj_id -> getProject proj_id
```

Comment:
This could probably also be shortened up with the bind operator, though you might get into trouble with operator precedence and need to use parens somewhere instead.  Something like this:
```
getProjectInfo project_name = do
    result <- runExceptT $ flip runSqlPool pool $ findProject project_name >>= getProject
    ...
```